include/spdlog/sinks/rotating_file_sink-inl.h

```diff
@@ -69,6 +69,12 @@ SPDLOG_INLINE filename_t rotating_file_sink<Mutex>::filename() {
     return file_helper_.filename();
 }
 
+template <typename Mutex>
+SPDLOG_INLINE void rotating_file_sink<Mutex>::rotate_now() {
+    SPDLOG_TRY { rotate_(); }
+    SPDLOG_CATCH_STD
```

Comment:
As a general rule, sinks do not swallow exception. The calling logger catches and handles them, so please remove the try/catch here

Reply:
Removed, as requested.

`SPDLOG_CATCH_STD` is defined in [common.h:118](https://github.com/gabime/spdlog/blob/15f539685bbb21b3bb01475ac89c6d8d53b5b60f/include/spdlog/common.h#L118).

Reply:
Thanks. Then don”t forget to catch it in your code then.